lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8101,6 +8123,47 @@ SwiftDeclConverter::makeSubscript(FuncDecl *getter, FuncDecl *setter) {
   return subscript;
 }
 
+VarDecl *
```

Comment:
Side note: `ImportDecl.cpp` is way too big, I'll move the decl synthesis logic into a separate file in another PR.